src/Http/Cookie/CookieInterface.php

```diff
@@ -114,12 +114,9 @@ public function getExpiry();
     /**
      * Get the timestamp from the expiration time
      *
-     * Timestamps are strings as large timestamps can overflow MAX_INT
-     * in 32bit systems.
-     *
-     * @return string|null The expiry time as a string timestamp.
+     * @return int|null The expiry time as a string timestamp.
```

Comment:
s/string/

Reply:
`The expiry time as an integer timestamp.` ?

Reply:
Or 'expiry time as an integer'